index.js

```diff
@@ -120,6 +121,7 @@ class TagInput<T> extends React.PureComponent<Props<T>, State> {
     inputColor: PropTypes.string,
     inputProps: PropTypes.shape(TextInput.propTypes),
     maxHeight: PropTypes.number,
+    minHeight: PropTypes.number,
```

Comment:
You're introducing a new prop here, but you haven't documented it in the `README`, and there's no corresponding Flow update for it. I think it's best to keep this new prop to a separate PR and limit this PR to just the fixes for the React lifecycle deprecation issue